src/vue-components/lookup.vue

```diff
@@ -42,38 +42,38 @@
       return {
         lookuptext: '',
         showLanguageSettings: false,
-        initLanguage: null,
-        currentLanguage: null,
         options: {},
-        resourceOptions: {}
+        resourceOptions: {},
+
+        currentLanguage: null
       }
     },
     props: {
       uiController: {
         type: Object,
         required: true
-      },
+      }/*,
       parentLanguage: {
         type: String,
         required: false
-      }
+      }*/
     },
     created: function () {
       if (this.uiController) {
-        this.options = this.uiController.options.clone(TempStorageArea)
-        this.resourceOptions = this.uiController.resourceOptions.clone(TempStorageArea)
-        if (this.parentLanguage) {
-          this.initLanguage = this.parentLanguage
-          this.currentLanguage = this.parentLanguage
-        } else {
-          this.currentLanguage = this.options.items.preferredLanguage.currentTextValue()
+        this.options = this.uiController.options
+        this.resourceOptions = this.uiController.resourceOptions
```

Comment:
I still think this should be a clone of the resourceOptions. Even if we aren't setting the short definition preferences in the panel right now, we will, and we don't want the changes made to them for single word lookups to override the overall preferences.